server/src/graphql/queries/repositories.ts

```diff
@@ -0,0 +1,131 @@
+import { Repository } from './../../generated/graphql';
+import fetch from 'node-fetch';
+import { QueryResolvers } from '../../generated/graphql';
+import jsonwebtoken from 'jsonwebtoken';
+import { config } from '../../config';
+import { prisma } from '../../prisma';
+export const repositories: QueryResolvers['repositories'] = async (
+  _,
+  __,
+  { userId }
+) => {
+  if (!userId) {
+    throw new Error('Unauthorized');
+  }
+
+  const user = await prisma.user.findUnique({
+    where: {
+      id: userId,
+    },
+  });
+
+  const payload = {
+    iat: Math.round(new Date().getTime() / 1000),
+    exp: Math.round(new Date().getTime() / 1000 + 60),
+    iss:
+      process.env.NODE_ENV === 'production'
+        ? config.githubAppId
+        : process.env.GITHUB_APP__ID,
+  };
+
+  //TODO FIGURE OUT A WAY TO PARSE THE PEM PROPERLY
+  const token = jsonwebtoken.sign(
```

Comment:
@pradel Note for this to-do do you have a clear idea what should be done to make this work w/o passing hardcoded key?